nvflare/private/fed/server/cmd_utils.py

```diff
@@ -214,15 +220,19 @@ def _process_replies_to_string(self, conn: Connection, replies) -> str:
         Returns:
             A string response.
         """
-        engine = conn.app_ctx
         response = "no responses from clients"
+        client_replies = {}
         if replies:
             response = ""
             for r in replies:
-                client_name = engine.get_client_name_from_token(r.client_token)
+                client_name = r.client_name
                 response += "client:" + client_name
                 if r.reply:
                     response += " : " + r.reply.body + "\n"
+                    client_replies[client_name] = r.reply.body
                 else:
                     response += " : No replies\n"
+                    client_replies[client_name] = MetaStatusValue.NO_REPLY
+
+        conn.add_meta({MetaKey.CLIENT_STATUS: client_replies})
```

Comment:
Most of the case we see:
```
meta = {MetaKey.STATUS: MetaStatusValue.xxx}
```
In here we specially see:
```
meta = {MetaKey.CLIENT_STATUS: {client1: MetaStatusValue.xxx, client2: MetaStatusValue.xxx}}
```

I am not sure if we want to mix the usage here.

Reply:
Different here. MetaKey.STATUS specifies the overall status of the request; whereas MetaKey.CLIENT_STATUS is a piece of meta information in the response.

Reply:
got it thanks